crates/ruma-common/tests/events/room_message.rs

```diff
@@ -308,96 +302,6 @@ fn markdown_content_serialization() {
     );
 }
 
-#[test]
-#[cfg(not(feature = "unstable-msc1767"))]
-fn relates_to_content_serialization() {
-    let message_event_content =
-        assign!(RoomMessageEventContent::text_plain("> <@test:example.com> test\n\ntest reply"), {
-            relates_to: Some(Relation::Reply {
-                in_reply_to: InReplyTo::new(
-                    event_id!("$15827405538098VGFWH:example.com").to_owned(),
-                ),
-            }),
-        });
-
-    let json_data = json!({
-        "body": "> <@test:example.com> test\n\ntest reply",
-        "msgtype": "m.text",
-        "m.relates_to": {
-            "m.in_reply_to": {
-                "event_id": "$15827405538098VGFWH:example.com"
-            }
-        }
-    });
-
-    assert_eq!(to_json_value(&message_event_content).unwrap(), json_data);
-}
-
-#[test]
-#[cfg(not(feature = "unstable-msc2676"))]
-fn edit_deserialization_061() {
-    let json_data = json!({
-        "body": "s/foo/bar",
-        "msgtype": "m.text",
-        "m.relates_to": {
-            "rel_type": "m.replace",
-            "event_id": "$1598361704261elfgc:localhost",
-        },
-        "m.new_content": {
-            "body": "bar",
-        },
-    });
-
-    let content = from_json_value::<RoomMessageEventContent>(json_data).unwrap();
-    assert!(content.relates_to.is_some());
-
-    let text = assert_matches!(
-        content.msgtype,
-        MessageType::Text(text) => text
-    );
-    assert_eq!(text.body, "s/foo/bar");
-    assert_matches!(text.formatted, None);
-}
-
-#[test]
-#[cfg(feature = "unstable-msc2676")]
-fn edit_deserialization_future() {
```

Comment:
This test looks like it should have been kept.

Reply:
Why? It looks a lot like the one in `relations.rs`.

Reply:
Oh, okay.